src/lib/remoteHelper.ts

```diff
@@ -202,11 +211,22 @@ const spawnPipedGitCommand = async (
             // remove inconsistent cache mark
             unsetCacheDirty(tmpPath, repo.dataTxId);
 
-            if (success)
+            if (success) {
+                await trackRepositoryUpdateEvent({
+                    repo_name: repo.name,
+                    repo_id: repo.id,
+                    result: 'SUCCESS',
+                });
                 log(`Successfully pushed repo '${repo.id}' to Protocol Land`, {
```

Comment:
thoughts on moving the tracking function call after the log in both if and else cases?

Reply:
Sure we can do that.